packages/front-end/components/PowerCalculation/PowerCalculationStatsEngineSettingsModal.tsx

```diff
@@ -24,13 +23,16 @@ export default function PowerCalculationStatsEngineSettingsModal({
     orgSettings.sequentialTestingTuningParameter ||
     DEFAULT_SEQUENTIAL_TESTING_TUNING_PARAMETER;
   const [currentParams, setCurrentParams] = useState(params);
+  const [currentEngine, setCurrentEngine] = useState<StatsEngineWithSequential>(
+    params.type
+  );
```

Comment:
Instead of two states, which is error prone, you can derive `currentEngine` from `currentParams`.

```ts
const currentEngine = currentParams.type === "bayesian" ? "bayesian" 
  : currentParams.sequentialTesting ? "sequential" 
  : "frequentist";
```